src/mono/sample/wasm/browser/runtime.js

```diff
@@ -2,19 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 "use strict";
-var Module = {
-    config: null,
+
+createDotnetRuntime(({ MONO, INTERNAL, BINDING, Module }) => ({
+    disableDotNet6Compatibility: true,
     configSrc: "./mono-config.json",
     onDotNetReady: function () {
         try {
-            App.init();
+            App.init({ MONO, INTERNAL, BINDING, Module });
```

Comment:
this seems like a bad pattern to encourage, it means that if we added a 5th namespace it would not get passed to App.init. The argument containing the namespaces should just have a name and that object should be passed through to App.init as-is

Reply:
yes, this will be better in ES6, where I would not have to do `.then()`.